gradle-guide-error-prone/src/main/java/com/palantir/gradle/guide/errorprone/types/GradleFix.java

```diff
@@ -61,8 +66,11 @@ public SuggestedFix render(GradleFixContext context, VisitorState state) {
 
         SuggestedFix.Builder fixBuilder = SuggestedFix.builder();
 
+        // Either getService() method or service field
+        String receiver = getServiceReceiver(context, state, fixBuilder);
```

Comment:
why not name this `serviceReceiver` so you're not having to do the weird `recv` further down